modules/ROOT/pages/apikit-tutorial.adoc

```diff
@@ -4,16 +4,100 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: apikit, raml, tutorial, api, rest
 
-In this procedure, you build and deploy an APIkit project that imports a RAML. You run the API in Anypoint Studio, use API Console to simulate calls to the API, prepare the project for deployment, deploy the project to Anypoint Platform, and apply a policy to the API.
+In this workflow, you build an APIkit project that imports a RAML. You run the API in Anypoint Studio and use API Console to simulate calls to the API.
 
 Studio 6.0 or later is recommended for this procedure. The API you create covers two main tasks:
 
 * Updating vending machine data: to update the central database on each sale
 * Tracking inventory: to find out if a machine is low on stock
 
-. xref:apikit-create.adoc[Create a new project].
-. xref:apikit-run-test.adoc[Run and test the project].
-. xref:apikit-add-logic.adoc[Add logic to the project].
-. xref:apikit-deploy.adoc[Deploy the project].
-. xref:apikit-apply-policy.adoc[Apply a policy].
-. xref:apikit-test-policy.adoc[Test a Policy].
\ No newline at end of file
+== Create a New APIkit Project
+
+This procedure launches the new project and generates a Main flow with an HTTP Listener, an APIkit Router, and references to flows to handle errors. The project support REST resources and methods. These flows, aptly called skeleton flows, are the bare minimum needed to support the RAML specification. The RAML file imported by this procedure resides in `src/main/api`.
+
+. Download the example file to your local drive:
++
+`+link:{attachmentsdir}/api.raml[api.raml]+`.
+. Under the File menu, select *New > Mule Project*.
+. In the New Mule Project wizard, enter a Name, myproject for example, Mule Runtime 3.8.0.
++
+. Check Add APIkit Components. In the RAML File field, browse to and select the downloaded `api.raml` file.
```

Comment:
**Add APIkit Components** and **RAML File** field. 

Technically this can be two steps unless the RAML field appearing is dependent on checking the APIkit Components box. In which case, just join with and. 
Check **Add APIkit Components**, and in the **RAML File** field...